exporters/FusionExporter/Source/AddIn/EUI-UIManagement.cpp

```diff
@@ -70,6 +70,80 @@ void EUI::prepareAllPalettes()
 	createGuidePalette();
 	createFinishPalette();
 	createProgressPalette();
+	createDriveWeightPalette();
+
+	 	// DEBUG PURPOSE = 
+}
+
+// Drivetrain Weight Palette
+
+bool EUI::createDriveWeightPalette() {
+	Ptr<Palettes> palettes = UI->palettes();
+	if (!palettes)
+		return false;
+
+	// Check if palette already exists
+	driveWeightPalette = palettes->itemById(K_DRIVE_WEIGHT_PALETTE);
+	if (!driveWeightPalette)
+	{
+		driveWeightPalette = palettes->add(K_DRIVE_WEIGHT_PALETTE, "Robot Drivetrain Weight Form", "Palette/dt_weight.html", false, true, true, 300, 150);
+		if (!driveWeightPalette)
+			return false;
+
+		driveWeightPalette->dockingState(PaletteDockingStates::PaletteDockStateRight);
+
+		addHandler<ReceiveFormDataHandler>(driveWeightPalette);
```

Comment:
A handler pointer should be provided as a second argument to `addHandler` (see `addHandler` calls for other forms)